exonum/src/runtime/dispatcher/mod.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
```

Comment:
Please reference the CLI command in `panic` messages in the `Dispatcher::block_on_migration` method; there's a FIXME there that should be removed in this PR.